invokeai/app/invocations/image.py

```diff
@@ -67,6 +67,29 @@ def invoke(self, context: InvocationContext) -> ImageOutput:
             image=ImageField(image_type=self.image_type, image_name=self.image_name)
         )
 
+class SaveImageInvocation(BaseInvocation):
```

Comment:
hrm... prefer not to let the user choose filenames. The filenames are currently selected to avoid conflicts (and overwriting results from other graphs). I could see this very quickly creating conflicts if users are sharing graphs.

Reply:
This _could_ potentially be a CLI command though instead of a node? That might be safer.